### DISCUSSION TITLE
how to update my pi adguard

### DISCUSSION THREAD

**Discussion by mayankgrathod** [AdguardTeam/AdGuardHome#4635] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by ainar-g** [comment]
If the automatic update failed for you for some reason, please follow the [manual on our FAQ Wiki page][1].

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update
